### PR TITLE
Bump Black and Flake8 deps

### PR DESCRIPTION
**Pull request by BeyondEvil**

### CODE REVIEW

**Comment by jkowalleck** [comment]
Don't know how hard you rely on having the latest dependencies(pipenv, npm) for the build env.

In case you bump those deps manualy often, you could have a bot that PRs dependency updates for [Keeping your dependencies updated automatically](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/keeping-your-dependencies-updated-automatically)
For example:  you could have this bot run scheduled(once a week) as a github action.

**Comment by BeyondEvil** [comment]
Good idea @jkowalleck 

Not sure dependabot supports updating pre-commit configs. But I'll check.

**Comment by BeyondEvil** [comment]
Seems there are open tickets for dependabot to support pre-commit:

https://github.com/dependabot/dependabot-core/issues/1524

https://github.com/dependabot/dependabot-core/issues/2040

Will hold off until that support is added.

**Comment by gnikonorov** [comment]
> Seems there are open tickets for dependabot to support pre-commit:
> 
> [dependabot/dependabot-core#1524](https://github.com/dependabot/dependabot-core/issues/1524)
> 
> [dependabot/dependabot-core#2040](https://github.com/dependabot/dependabot-core/issues/2040)
> 
> Will hold off until that support is added.

Noting that dependabot is being considered added here: https://github.com/pytest-dev/pytest-html/pull/419

File: Pipfile [hunk 1]
```diff
@@ -6,9 +6,10 @@ verify_ssl = true
 [dev-packages]
 pytest = "*"
 tox = "*"
-flake8 = "*"
-black = "*"
+flake8 = "==3.8.4"
```
Comment (gnikonorov) [inline Pipfile:9]: Can you add the following as a comment in the `Pipfile` so people aren't confused: https://github.com/pytest-dev/pytest-html/pull/414/files#r541766102

File: Pipfile [hunk 2]
```diff
@@ -6,9 +6,10 @@ verify_ssl = true
...
+black = "==20.8b1"
 pre-commit = "*"
+pytest-rerunfailures = "*"
```
Comment (gnikonorov) [inline Pipfile:12]: Shouldn't this be part of https://github.com/pytest-dev/pytest-html/pull/414 and not this PR?

**Review by gnikonorov — CHANGES_REQUESTED** [review]
One comment and one ask, but otherwise LGTM

File: Pipfile (hunk 2 above)
Comment (BeyondEvil) [reply Pipfile:12]: Indeed! Copypasta error on my part.

File: Pipfile (hunk 1 above)
Comment (BeyondEvil) [reply Pipfile]: Done.

File: Pipfile (hunk 2 above)
Comment (BeyondEvil) [reply Pipfile]: Done.

**Review by gnikonorov — APPROVED** [review]
🚀 ship it!

**Pull request merged**